spec.md

```diff
@@ -217,26 +217,41 @@ Incompatible changes to the schema SHOULD be reflected by a different URL.
   or application protocol's content-type metadata property. Normative rules
   for the binary mode and the content-type metadata mapping can be found 
   in the respective transport mapping specifications.
-    
+  
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
 ### extensions
+
+### idendityLabels
 * Type: `Map`
-* Description: This is for additional metadata and this does not have a
-  mandated structure. This enables a place for custom fields a producer or
-  middleware might want to include and provides a place to test metadata before
-  adding them to the CloudEvents specification.
-  See the [Extensions](extensions.md) document for a list of possible
-  attributes.
+* Description: This is a place for custom key-value pairs a producer or
+  middleware wants to include to provide additional identity information
+  about the event. The syntax and semantics of each specific label
+  inside this map are open for each event type producer to define,
```

Comment:
> for each event type producer to define

seems like in saying this, you have lost the value of the identity aspect if that is what this is for.

Reply:
We have to leave what goes inside this identity bag open since different types of events could have different labels and we can not predict all the labels. What we know is that an event could have an identity bag of labels, which serverless platforms (we can view the platform as an event consumer) need to support many serverless applications.